.travis.yml

```diff
@@ -42,11 +42,11 @@ script:
     then
       docker build --squash --no-cache=true -t ballerina/jre8:v1 ./base/base-image
       trivy --exit-code 1 --severity HIGH --no-progress ballerina/jre8:v1
-      mvn clean install -B -V
+      gradle build
```

Comment:
Need to pass in the following flags:
` --stacktrace -scan --console=plain --no-daemon`